app/src/main/java/lt/vilnius/tvarkau/SettingsActivity.kt

```diff
@@ -0,0 +1,114 @@
+package lt.vilnius.tvarkau
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.support.design.widget.Snackbar
+import android.support.v4.content.ContextCompat.getColor
+import android.widget.CompoundButton
+import android.widget.Toast
+import com.vinted.extensions.gone
+import com.vinted.extensions.visible
+import kotlinx.android.synthetic.main.settings_activity.*
+import lt.vilnius.tvarkau.ProblemsListActivity.MY_PROBLEMS
+import lt.vilnius.tvarkau.fragments.ReportImportDialogFragment
+import lt.vilnius.tvarkau.utils.SharedPrefsManager
+
+class SettingsActivity : BaseActivity(),
+        CompoundButton.OnCheckedChangeListener,
+        ReportImportDialogFragment.SettingsVilniusSignInListener {
+
+    private lateinit var prefsManager: SharedPrefsManager
```

Comment:
`lateinit` means, that value is not initialised at the moment of creation, but will be available before first use.
Used for injected properties.
Will throw exception if accessed before initialisation.